scss/_variables.scss

```diff
@@ -1337,8 +1339,8 @@ $form-range-thumb-box-shadow:              0 .1rem .25rem rgba($black, .1) !defa
 $form-range-thumb-focus-box-shadow:        null !default; // Boosted mod
 $form-range-thumb-focus-box-shadow-width:  $input-focus-width !default; // For focus box shadow issue in Edge
 $form-range-thumb-hover-bg:                var(--#{$prefix}highlight-bg) !default; // Boosted mod
-$form-range-thumb-active-bg:               $primary !default; // Boosted mod: instead of `tint-color($component-active-bg, 70%)`
-$form-range-thumb-active-border:           $primary !default; // Boosted mod
+$form-range-thumb-active-bg:               $brand-orange !default; // Boosted mod: instead of `tint-color($component-active-bg, 70%)`
```

Comment:
It's supposed to be dark orange in light mode and light orange in dark mode

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2362/commits/2d520357f26794adfde8ff77531e0401b5060690
We don't have the final colors for checkboxes and radios, so no modification for now